prob/torus_cbc/build.py

```diff
@@ -344,8 +345,8 @@
 print("Make sure to move the generated table to your run directory!")
 
 # Radiation units
-NUMIN_MEV = 1.0
-NUMAX_MEV = 100
+NUMIN_MEV = 0.1
+NUMAX_MEV = 500
```

Comment:
Did we want to keep this after all?

Reply:
No I just forgot to revert it. Will do now.

Reply:
Just make sure to change it in the nulib fork too